librz/diff/diff.c

```diff
@@ -146,12 +146,19 @@ static bool set_b(RzDiff *diff, const void *b, ut32 b_size) {
 	RzDiffMethodIgnore ignore = diff->methods.ignore;
 
 	/* we need to generate the hits list for B */
+	HtPPOptions opts = {
```

Comment:
Since you expose `HtPPOptions` please document the structure in the Ht header.

Reply:
It wasn't done by me. See ht_*_new_opt.